students/hayoung-gam/controllers/UserController.js

```diff
@@ -0,0 +1,39 @@
+import { UserService } from '../services';
+
+const getUsersList = async (req, res) => {
+  try {
+    const users = await UserService.getUsersList();
+
+    res.status(201).json({ users });
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+};
+
+const postSignUp = async (req, res) => {
```

Comment:
```suggestion
const signUp = async (req, res) => {
```
api 함수는 http 통신과는 독립적으로 만들어지는 것이기 때문에 함수 이름에 메소드를 명시하지는 않습니다 👍

Reply:
감사합니다. 코드를 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack-1-westarbucks-backend-team1/pull/5/commits/b5aaad5cda1c5bfb4387a70093156bdb94a0b9d3